### PR TITLE
[v6r22] Remove useless calls to gLogger.debug()

### PR DESCRIPTION
**Pull request by phicharp**
BEGINRELEASENOTES
CHANGE: remove many calls too gLogger.debug() is low level DB modules (in particular MySQL.py) that may slowdown services

For the moment only commented out the calls but if everybody agrees they should IMHO be removed...

Of course this should be done systematically in all services. This is just because I was looking at possible slowdown in JobDB handling which is very heavily called...
ENDRELEASENOTES

### CODE REVIEW

File: Core/Utilities/MySQL.py [hunk 1]
```diff
@@ -428,11 +428,11 @@ def _except(self, methodName, x, err):
     try:
       raise x
     except MySQLdb.Error as e:
-      self.log.debug('%s: %s' % (methodName, err),
+      #self.log.debug('%s: %s' % (methodName, err),
                      '%d: %s' % (e.args[0], e.args[1]))
```
Comment (fstagni) [inline Core/Utilities/MySQL.py]: Also this line needs to be commented out (https://travis-ci.org/DIRACGrid/DIRAC/jobs/598620679#L813)

And also please take care of the format: https://travis-ci.org/DIRACGrid/DIRAC/jobs/598620680

Comment (phicharp) [reply Core/Utilities/MySQL.py]: yes, thanks!

**Comment by atsareg** [comment]
Did you confirm that debug messages slow down the JobDB responses ?
With the introduction of centralized logging we should certainly review log messages, in particular in verbose and debug level in order to reduce the total number of messages but also redistribute messages in different levels. For example, now the verbose level is not really used as it mostly the same as the debug level. So, I would look at the problem from this point of view and not just suppress all the debug messages in MySQL.

**Comment by fstagni** [comment]
VERBOSE messages go to the centralized logging, DEBUG messages don't.

**Comment by atsareg** [comment]
DEBUG message are not going to the centralized logging now. Will it be always the case ?

**Comment by phicharp** [comment]
We have no quantitative measurement of the slow down due to messages. However, even when the log level is higher than VERBOSE, all calls make some formatting that takes time. 
As these printouts are anyway totally useless in a service that fills up already 10MB log files in a few minutes, better remove them.

**Pull request merged**